packages/fixie-common/src/agent.ts

```diff
@@ -287,10 +287,9 @@ export class FixieAgentBase {
           external: externalDeployment,
         },
         runtime: {
-          parametersSchema: runtimeParametersSchema,
+          parametersSchema: runtimeParametersSchema && JSON.stringify(runtimeParametersSchema),
```

Comment:
Is this the right thing? If you `JSON.stringify` a string, doesn't that wrap it in extra quotes we don't want here?